### ISSUE TITLE
Nuget package missing Web.Config key

### ISSUE THREAD

**Issue by medmondson** [alexbeletsky/elmah-mvc#89]
Fresh install of Elmah.MVC package.

/emah.axd is a 404

The following key is missing in the <handlers> section:

`<add name="Elmah" verb="POST,GET,HEAD" path="elmah.axd" type="Elmah.ErrorLogPageFactory, Elmah" />`

After adding it manually all is fine.

**Comment by medmondson** [comment]
More info http://stackoverflow.com/a/20789528/436028

**Comment by peterdew** [comment]
The goal of this project is to nót need /elmah.axd anymore. See
https://github.com/alexbeletsky/elmah-mvc#whats-the-goal

**Comment by cbaba20** [comment]
Yes, only <YourProjectURL>_/elmah_ is enough to view the error logs (check for in memory). 
Example : http:4327/elmah
